backend/telemetry/src/test/java/org/sonarsource/sonarlint/core/telemetry/TelemetryHttpClientTests.java

```diff
@@ -123,6 +123,14 @@ private void assertTelemetryUploaded(boolean isDebugEnabled) {
     telemetryLocalStorage.incrementFlightRecorderSessionsCount();
     telemetryLocalStorage.setMcpIntegrationEnabled(true);
     telemetryLocalStorage.setMcpTransportModeUsed(McpTransportMode.STDIO);
+    telemetryLocalStorage.setLabsJoined(true);
+    telemetryLocalStorage.setLabsEnabled(true);
+    telemetryLocalStorage.ideLabsLinkClicked("1");
```

Comment:
Let's have more realistic link & feature names to make it easier to understand the measure value in future. I would suggest for links 'changed_file_analysis_doc', 'privacy_policy', 'community', and for feedback 'agents_configuration', 'connected_mode', 'manage_dependency_risk'